src/components/unsigned-sweep.js

```diff
@@ -167,7 +168,7 @@ class UnsignedSweep extends Component {
         return obj;
       }, {});
 
-      if (this.state.coin === 'btc' && this.state.apiKey) {
+      if (this.state.coin === 'bsv' && this.state.apiKey) {
```

Comment:
mmm again did we just drop support for btc through these flows? I don't understand the mutation of this